PowerSyncExample/PowerSync/Schema.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import PowerSync
+
+let LISTS_TABLE = "lists"
+let TODOS_TABLE = "todos"
+
+let lists = Table(
+    name: LISTS_TABLE,
+    columns: [
+        // ID column is automatically included
+        Column(name: "name", type: ColumnType.text),
+        Column(name: "created_at", type: ColumnType.text),
+        Column(name: "owner_id", type: ColumnType.text)
+    ],
+    indexes: [],
+    localOnly: false,
+    insertOnly: false,
+    _viewNameOverride: LISTS_TABLE
```

Comment:
Right now default arguments are a hassle as mentioned here https://skie.touchlab.co/configuration/default-arguments. The underscore I assume is to avoid name collisions when bridging happens.

Reply:
👍